projects/com.oracle.truffle.llvm.nodes.impl/src/com/oracle/truffle/llvm/nodes/impl/cast/LLVMToI8Node.java

```diff
@@ -132,6 +132,11 @@ public byte executeI8(LLVM80BitFloat from) {
     public abstract static class LLVMAddressToI8Node extends LLVMToI8Node {
```

Comment:
Would it make sense to change the type of `fromNode` to `LLVMExpressionNode`? I think it is misleading to suggest that the node's input can always return an address, which I think, is, what using the node class should imply. I saw that  we still use the `LLVMAddressNode` at similar locations too.